webscraping/course-seq/storing/storer.js

```diff
@@ -1,30 +1,31 @@
 'use strict';
 
-let nodemailer = require('nodemailer');
-let remove = require("remove");
-let fs = require("fs");
-let MongoClient = require('mongodb').MongoClient;
-let assert = require('assert');
-let argv = require('minimist')(process.argv.slice(2));
-let Ajv = require('ajv');
-let ajv = new Ajv({
+const nodemailer = require('nodemailer');
+const remove = require("remove");
+const fs = require("fs");
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+const argv = require('minimist')(process.argv.slice(2));
+const Ajv = require('ajv');
+const ajv = new Ajv({
     "verbose": true,
     "allErrors": true
 });
 
-let validate = ajv.compile(JSON.parse(fs.readFileSync('../json-schema/recommendedSequence.json', 'utf8')));
-let mongoServerUrl = 'mongodb://138.197.6.26:27017/';
-let devDbName = "courseplannerdb-dev";
-let prodDbName = "courseplannerdb";
-let dbName = (argv.prod) ? prodDbName : devDbName;
-let dbFullUrl = mongoServerUrl + dbName;
+const validate = ajv.compile(JSON.parse(fs.readFileSync('recommendedSequence.json', 'utf8')));
+const mongoServerUrl = 'mongodb://138.197.6.26:27017/';
+const devDbName = "courseplannerdb-dev";
+const prodDbName = "courseplannerdb";
+const dbName = (argv.prod) ? prodDbName : devDbName;
```

Comment:
Why all the constants instead of using `let`? :thinking:

Reply:
I just find that it expresses 'programmer intent' better.  `const`s throw an error if you try to assign a new value to them.  So, when you see a `const`,  it's clear that the variable name will always refer to the same object.  The `const` keyword was made specifically for clarity, since why else would you want a keyword that prevents re-assignment.